integration/Cargo.toml

```
@@ -23,8 +23,8 @@ sp-core = "26.0"
 sp-runtime = "29.0"
 sp-keyring = "29.0"
 
-polymesh-api = { version = "3.7.0", features = ["download_metadata"] }
-polymesh-api-tester = { version = "0.3.1" }
+polymesh-api = { version = "3.9.1", features = ["download_metadata"] }
+polymesh-api-tester = { version = "0.8.0" }
```

```suggestion
polymesh-api-tester = { version = "0.8.0", default-features = false, features = ["download_metadata", "polymesh_v7"] }
```
We don't have `serde` support for `AssetId` right now.  I think a quick fix is to disable default features.

---

Seems there is an issue with optional `serde` support.  I have published a fix to `polymesh-api-client` (version 3.8.1) to add serde support.  Just update the  deps `cargo update -p polymesh-api-client`.